Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
test-full does as of now the same as test-verbose, since we discussed that the future command should be full, delete verbose or at least explain me why its still here